app/inputs/concerns/array_aware_input.rb

```diff
@@ -65,7 +65,7 @@ def array_of_inputs(wrapper_options = {})
   end
 
   def include_field_template(element_fields, array_options)
-    element_field_template = CGI.escapeHTML(element_fields.first.gsub(%(_0"), %(_[[index]]")))
+    element_field_template = CGI.escapeHTML(element_fields.first.gsub(%(_0"), %(_[[index]]")).gsub(/value=".*?"/, ' value=""'))
```

Comment:
The regex needs a space at the start, and there is no need to replace if the attribute is already blank.

Could we therefore use the alternative I suggested in a previous comment, i.e. `.gsub(/ value="[^"]+"/, ' value=""'))`